lib/types/query/existence.ts

```diff
@@ -19,9 +19,9 @@ export type ExistenceFilter<
 >
 
 /**
- * @name exists
- * @desc check for existence
- * @see [Documentation]{@link https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters/existence}
+ * Check for existence in provided fields of an entry
```

Comment:
since the filter is `${fieldName}[exists]: true` I would say it's check for the provided field's existence in an Entry